src/styles/partials/components/_section.scss

```diff
@@ -1,58 +1,64 @@
 .dg_section {
-	&.dark {
-		background: $pale-blue;
-		color: $white;
-
-		h2 {
-			@include alt-h2-heading($gold);
-		}
-
-		p {
-			color: $white;
-		}
-
-		.dg_button {
-			background: none;
-			color: $white;
-			border-color: $white;
-
-			&:hover,
-			&.hovered {
-				background: $blue-lightest;
-			}
-
-			&:active,
-			&.active,
-			&:disabled,
-			&.disabled {
-				background: none;
-			}
-
-			&:disabled,
-			&.disabled {
-				border-color: $gray-light;
-			}
-		}
-
-		.dg_button-link {
-			&:hover,
-			&.hovered {
-				background: none;
-				color: $blue-lightest;
-			}
-
-			&:active,
-			&.active {
-				color: $white;
-			}
-		}
-
-		.dg_button,
-		.dg_button-link {
-			&:disabled,
-			&.disabled {
-				color: $gray-light;
-			}
-		}
-	}
-}
+    &.light {
+        background: $gray-lightest;
+    }
+
+    &.dark {
+        background: $pale-blue;
+        color: $white;
+
+        h2 {
+            @include alt-h2-heading($gold);
+        }
+
+        p {
+            color: $white;
+        }
+
+        .dg_button {
```

Comment:
Consider organizing css properties in alphabetical order for consistency in the source.

Reply:
addressed

Reply:
Looks good.